src/main/java/spoon/Launcher.java

```diff
@@ -75,7 +75,7 @@
 
 	public static final String OUTPUTDIR = "spooned";
 
-	private final Factory factory;
+	protected Factory factory;
```

Comment:
Is it really necessary to make the factory protected?

Reply:
I can replace that with `super(oldFactory)`, but I would like to avoid undesirable call of `processArguments()` in the `Launcher` constructor. Alternatively, I can add another protected constructor to the `Launcher`, which does not call `processArguments()`.